Gemfile.lock

```diff
@@ -8,22 +8,20 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.7.0)
-      public_suffix (>= 2.0.2, < 5.0)
-    apparition (0.1.0)
-      backports
-      capybara (~> 3.12, < 4)
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
+    apparition (0.6.0)
```

Comment:
Had to update apparition. There was some interaction between the newer versions of chrome that just caused some tests to hang indefinitely.